packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import styles from './Card.module.css';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+
+export function RaisedLabel({
+  trending,
+  listMode,
+  type,
+}: {
```

Comment:
Let's introduce an interface in this same file for these properties, just to tidy it up a bit.